### PR TITLE
[Doc] add 3.2.3 features

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/en/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md [hunk 1]
```diff
@@ -1107,3 +1108,143 @@ WITH BROKER
...
+PROPERTIES
+(
+    "strip_outer_array" = "true",
+    "jsonpaths" = "[\"$.category\",\"$.price\",\"$.author\"]"
+);
+```
+
+> **NOTE**
+>
+> In the preceding example, the outermost layer of the JSON data is an array structure as indicated by a pair of square brackets `[]`. The array structure consists of multiple JSON objects that each represent a data record. Therefore, you need to set `strip_outer_array` to `true` to strip the outermost array structure. The key **title** that you do not want to load is ignored during loading.
```
Comment (evelynzhaojie) [inline docs/en/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md:1207]: each represents

File: docs/en/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md [hunk 2]
```diff
@@ -1107,3 +1108,143 @@ WITH BROKER
...
+    "comments": ["3 records", "there will be 3 rows"]
+}
+```
+
+To load only `category`, `author`, and `price` from `example3.json`, run the following command:
+
+```SQL
+LOAD LABEL LOAD LABEL test_db.label17
+(
+    DATA INFILE(""hdfs://<hdfs_host>:<hdfs_port>/user/starrocks/data/input/example3.csv"")
```
Comment (evelynzhaojie) [inline docs/en/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md:1230]: two pairs of double quotation marks, remove

File: docs/en/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md [hunk 3]
```diff
@@ -1107,3 +1108,143 @@ WITH BROKER
...
+(
+    "json_root"="$.RECORDS",
+    "strip_outer_array" = "true",
+    "jsonpaths" = "[\"$.category\",\"$.price\",\"$.author\"]"
+);
+```
+
+> **NOTE**
+>
+> In the preceding example, the outermost layer of the JSON data is an array structure as indicated by a pair of square brackets `[]`. The array structure consists of multiple JSON objects that each represent a data record. Therefore, you need to set `strip_outer_array` to `true` to strip the outermost array structure. The keys `title` and `timestamp` that you do not want to load are ignored during loading. Additionally, the `json_root` parameter is used to specify the root element, which is an array, of the JSON data.
```
Comment (evelynzhaojie) [inline docs/en/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md:1250]: each represents

File: docs/zh/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md [hunk 4]
```diff
@@ -1159,3 +1160,143 @@ WITH BROKER
...
+
+2. （可选）根据 `json_root` 参数匹配 JSON 数据的根节点。
+
+    > **说明**
+    >
+    > 仅在 JSON 数据存在根节点时涉及，需要通过 `json_root` 参数来指定根节点。
+
+3. 根据 `jsonpaths` 参数提取待导入的 JSON 数据。
+
+##### 不指定 JSON 根节点、使匹配模式导入数据
```
Comment (evelynzhaojie) [inline docs/zh/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md:1223]: 使用

File: docs/zh/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md [hunk 5]
```diff
@@ -1159,3 +1160,143 @@ WITH BROKER
...
+    {"category":"xuxb111","author":"1avc","title":"SayingsoftheCentury","price":895},
+    {"category":"xuxb222","author":"2avc","title":"SayingsoftheCentury","price":895},
+    {"category":"xuxb333","author":"3avc","title":"SayingsoftheCentury","price":895}
+]
+```
+
+您可以通过指定 `jsonpaths` 参数进行精准导入，例如只导入 `category`、`author`、`price` 三个字段的数据：
+
+```SQL
+LOAD LABEL LOAD LABEL test_db.label16
```
Comment (evelynzhaojie) [inline docs/zh/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md:1238]: LOAD LABLE 和下面的 hdfs 路径双引号

File: docs/zh/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md [hunk 6]
```diff
@@ -1159,3 +1160,143 @@ WITH BROKER
...
+    "strip_outer_array" = "true",
+    "jsonpaths" = "[\"$.category\",\"$.price\",\"$.author\"]"
+);
+```
+
+> **说明**
+>
+> 上述示例中，JSON 数据的最外层是一个通过中括号 [] 表示的数组结构，并且数组结构中的每个 JSON 对象都表示一条数据记录。因此，需要设置 `strip_outer_array` 为 `true`来裁剪最外层的数组结构。导入过程中，未指定的字段 `title` 会被忽略掉。
+
+##### 指定 JSON 根节点、使匹配模式导入数据
```
Comment (evelynzhaojie) [inline docs/zh/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md:1261]: 使用

File: docs/en/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md (hunk 3 above)
Comment (amber-create) [reply docs/en/sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md:1250]: 这里应该用复数，对齐的应该是主语 objects that

**Review by EsoragotoSpirit — DISMISSED** [review]

**Review by evelynzhaojie — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Pull request merged**